cnf-tests/hack/compare-gen-md.sh

```diff
@@ -4,8 +4,9 @@
 
 MDFILE="TESTLIST.md"
 
-mv cnf-tests/${MDFILE} _cache/${MDFILE}.old
+mv ${MDFILE} _cache/${MDFILE}.old
```

Comment:
this fails if cnf-tests/_cache was not created yet
should we add mkdir -p _cache before?

Reply:
Right, not sure why I did not catch it.
Probably the path was created automatically by ginkgo or something.
Added it to the local common.sh file plus added the folder to gitignore